### PR TITLE
arman.is-a.dev

### PR DESCRIPTION
**Pull request by ItzArman09**

### CODE REVIEW

File: domains/arman.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Arman's personal website!",
+  "repo": "https://github.com/ItzArman09/website",
+  "owner": {
+    "username": "ItzArman09",
+    "email": "gyfufifa@norwegischlernen.info"
+  },
+  "record": {
+    "CNAME": "https://itzarman09.github.io/website"
```
Comment (MaskDuck) [inline domains/arman.json:9]: no slash on cname

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/arman.json (hunk 1 above)
Comment (ItzArman09) [reply domains/arman.json:9]: oh my bad wait

Comment (ItzArman09) [reply domains/arman.json]: done, edited

File: domains/arman.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Arman's personal website!",
+  "repo": "https://github.com/ItzArman09/website",
+  "owner": {
+    "username": "ItzArman09",
+    "email": "gyfufifa@norwegischlernen.info"
+  },
+  "record": {
+    "CNAME": "arman.is-a.dev"
```
Comment (MaskDuck) [inline domains/arman.json:9]: this should be points to your host, not your domain itself.

see your host's custom domain instruction for more info.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/arman.json (hunk 2 above)
Comment (ItzArman09) [reply domains/arman.json:9]: uhm, im using github pages

Comment (ItzArman09) [reply domains/arman.json]: edited, check now

Comment (ItzArman09) [reply domains/arman.json]: image
whats this?

**Review by MaskDuck — APPROVED** [review]
nice

File: domains/arman.json (hunk 2 above)
Comment (MaskDuck) [reply domains/arman.json]: the pr didnt get merged

a maintainer would need to merge before dns record get added

Comment (ItzArman09) [reply domains/arman.json]: okay thank you

**Pull request merged**